test/lib/sdkdriver/src/transport/index.d.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * A {@link Transport} represents an interface for communicating
+ * between the mocha.js process that drives a cross-browser test
```

Comment:
This is fine for now, but in the future I think we can be more generic. Although we'll initially use Mocha, it's not strictly necessary and this could be used with other things.